src/Ushahidi/Modules/V5/Commands/CreateCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Ushahidi\Modules\V5\Commands;
+
+use Illuminate\Database\Eloquent\Model;
+use App\Bus\Command\Command;
+
+class CreateCommand implements Command
```

Comment:
I'd remove that. Queries and Commands should be specific, generalisation in this case will lead to more errors in the future.